distributed/scheduler.py

```diff
@@ -512,7 +525,8 @@ def clean(self):
             extra=self._extra,
         )
         ts: TaskState
-        ws._processing = {ts._key: cost for ts, cost in self._processing.items()}
+        ws._processing = {ts.key: cost for ts, cost in self._processing.items()}
+        ws._executing = {ts.key: duration for ts, duration in self._executing.items()}
```

Comment:
This fetches the C-level attributes of `TaskState` when using Cython, which can be quite a bit faster.

```suggestion
        ws._processing = {ts._key: cost for ts, cost in self._processing.items()}
        ws._executing = {ts._key: duration for ts, duration in self._executing.items()}
```

Reply:
Thanks @jakirkham!